network-api/networkapi/wagtailpages/templates/wagtailpages/blog_index_page.html

```diff
@@ -22,6 +22,15 @@
   {% endif %}
 {% endblock %}
 
+{% block featured_video_post %}
+  {% if not filtered %}
+    {%  for video_post in page.specific.featured_video_post.all %}
```

Comment:
@tbrlpld Drawing inspiration from a previous ticket that we worked together on, instead of using an if statement I am just looping through the list, even though it is limited to 1 choice. So if there is not one set in the CMS, this field wont render at all